<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LauncherTab</class>
 <widget class="QWidget" name="LauncherTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>Logger</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="logNumber_label">
          <property name="text">
           <string>Log File Limit:</string>
          </property>
          <property name="buddy">
           <cstring>logNumber_spinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="logTip_label">
          <property name="text">
           <string>No Limit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="logNumber_spinBox">
          <property name="minimum">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lang_label">
       <property name="text">
        <string>Language:</string>
       </property>
       <property name="buddy">
        <cstring>lang_cb</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lang_cb">
       <property name="whatsThis">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>507</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
